pkg/process/config/config.go

```diff
@@ -68,11 +67,10 @@ var (
 	processChecks   = []string{ProcessCheckName, RTProcessCheckName}
 	containerChecks = []string{ContainerCheckName, RTContainerCheckName}
 
-	moduleCheckMap = map[sysconfig.ModuleName][]string{
-		sysconfig.NetworkTracerModule:        {ConnectionsCheckName, NetworkCheckName},
-		sysconfig.OOMKillProbeModule:         {OOMKillCheckName},
-		sysconfig.TCPQueueLengthTracerModule: {TCPQueueLengthCheckName},
-		sysconfig.ProcessModule:              {ProcessModuleCheckName},
+	// ModuleCheckMap
+	ModuleCheckMap = map[sysconfig.ModuleName][]string{
+		sysconfig.NetworkTracerModule: {ConnectionsCheckName},
```

Comment:
OOMKillProbeModule and TCPQueueLengthCheckName aren't used anywhere, so I went ahead and removed them.